backend/es/es_index.py

```diff
@@ -93,20 +99,26 @@ def create(client: Elasticsearch, corpus: Corpus, add: bool = False, clear: bool
             settings=settings,
             mappings=es_mapping,
         )
+        return index_name
```

Comment:
If you're using the `--add` flag (to add documents to an existing index), this function won't reach this line, but `return None` on line 69. I expect that will cause an issue when `populate()` is called with `versioned_index_name = None`.

This should be straightforward to fix: just return the name of the existing index in that case.

Reply:
Yes, and luckily, this is already caught by a unit test. :-)